connectivity/check/policy.go

```diff
@@ -32,6 +32,11 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+const (
+	// testNamespacePlaceholder will get replaced with the test namespace in use when parsing CPNs from YAML.
+	testNamespacePlaceholder = "${TEST_NAMESPACE}"
```

Comment:
I'm not a fan of the placeholder idea, that will create complexity sprawl in the long run.

My idea was to have policies in yaml form with e.g. the namespace field empty and populate them in Go during execution. (empty fields to make sure we don't inadvertently clobber them) This would mean these policies can no longer be applied using `kubectl`, but that's not the case with this approach either.

Reply:
> I think a solution to this problem might require some more thought.

That's why I was proposing this as a quick, interim fix until we figure it out. 🙂

> This would mean these policies can no longer be applied using kubectl, but that's not the case with this approach either.

`cat connectivity/manifests/client-egress-to-echo.yaml| TEST_NAMESPACE=foo envsubst | k apply -f-` works 🙂 Of course it isn't as simple as just `kubectl`ing, but it does actually work.